Controller/Redirect/Process.php

```diff
@@ -301,6 +301,7 @@ public function execute()
                         )
                     );
                     $this->logger->addDebug(__METHOD__ . '|5|');
+                    $this->secondChance();
                     return $this->_redirect('/');
                     // $this->redirectFailure();
```

Comment:
what was redirectFailure doing that it was impacting pending orders?

Reply:
Sorry, it was old commit, was cleared